product_multi_link/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2011 Guewen Baconnier,Camptocamp,Elico-Corp,Sodexis
```

Comment:
Put a line for each contributor and its year. For example, you are making the contribution on 2016, so you have to put:
```
# © 2016 Sodexis
```